src/VSCodeExtension/src/generator-qsharp/.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node_js
+node_js:
+  - v14
+  - v12
+  - v10
+  - v8
+  - v6
+  - v4
+after_script: cat ./coverage/lcov.info | coveralls
```

Comment:
The `generator-generator` template makes this for use with Travis CI, but we use a mix of Azure Pipelines and GitHub Actions for our CI; I'd suggest dropping this file and creating an equivalent pipeline step or GitHub action instead.